flake.nix

```diff
@@ -37,6 +43,27 @@
         melange.overlays.default
         ocamllsp.overlays.default
       ];
+      use-this-dune-static = self: super: {
+        ocaml-ng =
+          super.ocaml-ng // {
+            ocamlPackages_4_14 = super.ocaml-ng.ocamlPackages_4_14.overrideScope (pself: psuper: {
```

Comment:
is there a simpler way to do this override?

Reply:
maybe `ocamlPackages = super.ocaml-ng.ocamlPackages_4_14.overrideScope ...` and then use `pkgs.ocamlPackages` throughout?